snapfaas/bins/multivm/main.rs

```diff
@@ -109,6 +110,16 @@ fn main() {
     // kick off the resource manager
     let manager_handle = manager.run();
 
+    thread::spawn(|| {
+        loop {
+            thread::sleep(time::Duration::from_secs(5));
+            fs::utils::taint_with_label(labeled::buckle::Buckle::top());
+            let mut fs = fs::FS::new(&*snapfaas::labeled_fs::DBENV);
+            let n = fs.collect_garbage().unwrap();
+            log::debug!("garbage collected {}", n)
+        }
+    });
```

Comment:
Given that multivm is a local machine manager, I think the thread should be put in the sheduler for now. And when storage is made remote, we can move it to the storage server.

Reply:
Sounds reasonable. Fixed